### DISCUSSION TITLE
Command 'nats-server' not found

### DISCUSSION THREAD

**Discussion by khanakia** [nats-io/nats-server#4838] [Q&A]
I am trying to install nats server on Ubuntu but getting below error after installing

```sh
root@ip-172-31-9-170:/# go install github.com/nats-io/nats-server/v2@latest
root@ip-172-31-9-170:/# nats-server

Command 'nats-server' not found, but can be installed with:

snap install nats-server

```

**Comment by khanakia** [comment]
I figured it installed the NATS server in this directory `/root/go/bin/nats-server` not sure which but when I run the below command to get the go directory it shows different path

```
root@ip-172-31-9-170:/tmp# which go
/usr/local/go/bin/go
```

**Comment by ripienaar** [comment]
The go install method is really only for go developers. 

Sounds like you would have a better time downloading binaries or packages from our release page

**Reply by khanakia** [reply]
Ok but there is no other method specified in the docs on how to install on the Ubuntu server ?
 https://docs.nats.io/running-a-nats-service/introduction/installation

**Reply by ripienaar** [reply]
https://docs.nats.io/running-a-nats-service/introduction/installation#downloading-a-release-build

Above installing from source. Installing from source should be your last resort.

**Reply by khanakia** [reply]
Ok thanks
